TASVideos/Pages/Shared/_Layout.cshtml

```diff
@@ -27,6 +27,7 @@
 	<link rel="alternate" href="/publications.rss" type="application/rss+xml" title="TASVideos RSS Publication Feed">
 	<link rel="alternate" href="/wiki.rss" type="application/rss+xml" title="TASVideos RSS Wiki Feed">
 	<partial name="_MetaTags" />
+ 	<script>document.head.appendChild(document.createElement("style")).innerHTML = '.d-none-js { display: none !important; }';</script>
```

Comment:
Alternatively:
```suggestion
	<!-- can move to main stylesheet --><style>:root.supports-js .d-none-js { display: none !important; }</style>
	<script>document.documentElement.classList.add("supports-js");</script>
```

Reply:
(I can't test right now) Have you tested whether that prevents the flashing on repeated reloading on the usual browsers on desktop and mobile?
It may look the same, but I noticed adding a <style> blocks cached rendering better than adding a css class.

Reply:
I haven't (I also can't test atm)

Reply:
they are all a single taghelper now if that helps with a better strategy.  But can we close this? we dont' want this hacky approach

Reply:
Yeah I'm going to close this PR. This seems to need a different approach altogether.